### PR TITLE
Update config.json

### PR DESCRIPTION
**Pull request by tarrenj**

### CODE REVIEW

**Comment by tarrenj** [comment]
Must also merge: https://github.com/ZencashOfficial/botio/pull/2

File: config.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "private_repo": "data",
```
Comment (cronicc) [inline config.json]: change to `"private_repo": true,`

File: config.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
   "repo": "ZencashOfficial/website",
```
Comment (cronicc) [inline config.json]: Might as well change to the private repo already.

Comment (tarrenj) [reply config.json]: Fixed.

File: config.json (hunk 1 above)
Comment (tarrenj) [reply config.json]: Fixed.

**Pull request merged**